Lesson 2/src/index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Курсы компании "Гарцующий пони"</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="style.css">  
+</head>
+<body>
+    <header>
+      <nav>
+        <ul>
+          <li><a href="Home">Главная</a></li>
+          <li><a href="Blog">Блог</a></li>
+          <li><a href="Contacts">Контакты</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <h1>Курсы компании "Гарцующий пони"</h1>
+      <div class="wrapper">
+        <article class="card">
+          <header class="card-header">
+            <h2>SEO для начинающих</h2>
+            <p>Иван Иванович</p>
+          </header>
+          <div class="main-content">
+            <img src="./assets/img/image1.png" alt="seo">
+            <p>Благодаря этому курсу вы научитесь задавливать конкурентов с помощью ссылочной массы, а не качественного
+              контента.</p>
+          </div>
+          <footer class="card-footer">
+            <p>13 февраля 2023</p>
+          </footer >
+        </article>
+        <article class="card">
+          <header class="card-header">
+            <h2>WordPress разработка</h2>
+            <p>Ольга Ольгина</p>
+          </header>
+          <div class="main-content">
+            <img src="./assets/img/image2.png" alt="worldpress-course">
```

Comment:
```suggestion
            <img src="./assets/img/image2.png" alt="Wordpress course">
```

в `alt` можно использовать и пробелы, и заглавные - здесь названия когда картинка не загрузилась и то, что зачитает читалка (тире не нужны)